draft-ietf-mls-protocol.md

```diff
@@ -924,16 +882,11 @@ proceeds as shown in the following diagram:
 Update Secret -> HKDF-Extract = Epoch Secret
                      |
                      |
-                     +--> Derive-Secret(., "msg", ID, Epoch, Msg)
+                     +--> Derive-Secret(., "msg", GroupState_n)
```

Comment:
If we don't use the `_[N]` subscript to give the precise version of the secrets, maybe just use `GroupState` instead of `GroupState_n` ?